strawberry/schema/schema_converter.py

```diff
@@ -480,3 +471,25 @@ def from_union(self, union: StrawberryUnion) -> GraphQLUnionType:
         )
 
         return graphql_union
+
+    def _get_is_type_of(
+        self,
+        object_type: TypeDefinition,
+    ) -> Optional[Callable[[Any, GraphQLResolveInfo], bool]]:
+        if object_type.is_type_of:
+            return object_type.is_type_of
+
+        if object_type.interfaces:
+
+            def is_type_of(obj: Any, _info: GraphQLResolveInfo) -> bool:
+                if object_type.concrete_of and (
+                    hasattr(obj, "_type_definition")
+                    and obj._type_definition.origin is object_type.concrete_of.origin
+                ):
+                    return True
```

Comment:
@BryceBeagle any suggestion for improving this? :)

Reply:
Hmm... 

Since this is only used in one specific place, I would probably _not_ make this a method on the class, instead making it another nested function in `from_object`.

Other than that, I'm not sure of anything else that could be done.

Reply:
moved :)